widgets/pagination.js

```diff
@@ -24,9 +25,11 @@ function pagination({
     labels,
     maxPages,
     showFirstLast,
-    hideWhenNoResults = true
+    hideWhenNoResults = true,
+    scrollTo = 'body'
   }) {
   var containerNode = utils.getContainerNode(container);
+  var scrollToNode = scrollTo !== false ? utils.getContainerNode(scrollTo) : false;
```

Comment:
The documentation suggest that I could use a boolean here. So yes, I would make `true` the same as the default.